.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -23,16 +28,42 @@ A clear and concise description of what you expected to happen.
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
+**Zilla Environment:**
+<!--
+Describe the Host environment including:
+- Zilla start command
+- Environment variables
+
+Describe a docker container:
+```
+docker inspect <zilla container name> | jq -r '.[].Config' 
+```
+
+Describe a k8s pod:
+```
+kubectl describe pod <zilla pod name prefix> -n <zilla k8s namespace>
+```
+-->
+
+**Attach the `zilla.yaml` config file:**
+
+**Attach the `zilla dump` pcap file:**
+<!-- docs: https://docs.aklivity.io/zilla/latest/reference/config/zilla-cli.html#zilla-dump -->
+```
+ZILLA_INCUBATOR_ENABLED=true ./zilla dump --verbose --write /tmp/zilla.pcap
+```
```

Comment:
Perhaps we need a line to tell them to execute this from inside the docker container?